frontend/src/features/account/LoginLoading.tsx

```diff
@@ -0,0 +1,41 @@
+import './Login.scss';
+import React from 'react';
+import { Container, Row, Col, Button } from 'react-bootstrap';
+import PIMSlogo from 'assets/images/PIMSlogo/logo_with_text.png';
+import FilterBackdrop from 'components/maps/leaflet/FilterBackdrop';
+
+/**
+ * Display a placeholder of the PIMS login screen when keycloak is being initialized.
+ */
+const LoginLoading = () => {
```

Comment:
this is a "fake" version of the login page that is not interactive, this displays when the app is initializing.